build-system/tasks/visual-diff/index.js

```diff
@@ -56,8 +53,6 @@ const WEBSERVER_TIMEOUT_RETRIES = 10;
 const NAVIGATE_TIMEOUT_MS = 30000;
 const MAX_PARALLEL_TABS = 5;
 const WAIT_FOR_TABS_MS = 1000;
```

Comment:
I noticed an error in the logs that might have to do with parallelism. 
![image](https://user-images.githubusercontent.com/26553114/67249772-d924f500-f436-11e9-8657-e0c1e4ae9bf4.png)

Do any of these values need to change?

Reply:
This is an error in the agent (I managed to isolate it locally) - once we figure out the rest we can report an error to the https://github.com/percy/percy-agent repo

Reply:
Reported here: https://github.com/percy/percy-agent/issues/402